### PR TITLE
[13.0][MIG] account_invoice_line_report: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
TT25936

ping @carlosdauden @pedrobaeza

### CODE REVIEW

File: account_invoice_line_report/report/account_invoice_report_view.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            </tree>
+        </field>
+    </record>
+    <record id="view_account_invoice_report_search" model="ir.ui.view">
+        <field name="model">account.invoice.report</field>
+        <field name="inherit_id" ref="account.view_account_invoice_report_search" />
+        <field name="arch" type="xml">
+            <field name="invoice_date" position="before">
+                <field
+                    name="product_uom_id"
```
Comment (joao-p-marques) [inline account_invoice_line_report/report/account_invoice_report_view.xml]: I have some doubts about using this variable here. In v12, it used `uom_name`, which was removed in v13. I think this is doing the correct mapping, but I also don't understand very well the search filter it is doing below (why compare `partner_id` with `uom_name`?) so I'm not sure if it is doing all the functionality from the previous version...

File: account_invoice_line_report/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2018 Vicent Cubells - Tecnativa <vicent.cubells@tecnativa.com>
```
Comment (victoralmau) [inline account_invoice_line_report/__manifest__.py]:
```suggestion
# Copyright 2017 Tecnativa - Carlos Dauden
# Copyright 2018 Tecnativa - Vicent Cubells
```

File: account_invoice_line_report/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Dauden <carlos.dauden@tecnativa.com>
+  * Vicent Cubells <vicent.cubells@tecnativa.com>
```
Comment (victoralmau) [inline account_invoice_line_report/readme/CONTRIBUTORS.rst]:
```suggestion
  * Carlos Dauden
  * Vicent Cubells
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job!

**Comment by joao-p-marques** [comment]
Thank you for the review @victoralmau 
Changes made.

**Review by victoralmau — APPROVED** [review]

File: account_invoice_line_report/report/account_invoice_report_view.xml (hunk 1 above)
Comment (carlosdauden) [reply account_invoice_line_report/report/account_invoice_report_view.xml]: This is a trick to filter by "on fly related field"... Now we can use https://github.com/OCA/server-ux/tree/13.0/base_search_custom_field_filter so you can remove this filter field and the next.

**Review by carlosdauden — APPROVED** [review]

File: account_invoice_line_report/report/account_invoice_report_view.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                    string="Partner Tags"
+                    filter_domain="[('partner_id.commercial_partner_id.category_id', '=', self)]"
+                />
+                <field
+                    name="product_uom_id"
```
Comment (yostashiro) [inline account_invoice_line_report/report/account_invoice_report_view.xml]: I suppose the name should be changed for this field and above?

**Comment by joao-p-marques** [comment]
Changes made @carlosdauden

File: account_invoice_line_report/report/account_invoice_report_view.xml (hunk 4 above)
Comment (joao-p-marques) [reply account_invoice_line_report/report/account_invoice_report_view.xml]: Following @carlosdauden's suggestion, I ended up removing both filters.

**Review by victoralmau — APPROVED** [review]

**Pull request merged**